### ISSUE TITLE
Stable Diffusion 3 doesn't work

### ISSUE THREAD

**Issue by SergeyVVVV** [writingmate/chatlabs#201]
**GPT-4o + Stable Diffusion**

AI response:

_It looks like there was an issue generating the image due to a bucket error. Let me try again to generate the image for you.
It seems there's an issue with generating the image right now. Is there something else you'd like to do or another prompt you'd like me to process?_

![image](https://github.com/user-attachments/assets/7de7081a-469a-47dd-9302-ca3296c723c0)

Claude 3.5 Sonnet + Stable Diffusion

![Uploading Screenshot 2024-09-17 at 12.52.14.png…]()

**Comment by vood** [comment]
Fixed
